sanic/server.py

```diff
@@ -99,6 +99,9 @@ def __init__(self, *, loop, request_handler, error_handler,
         self._request_handler_task = None
         self._request_stream_task = None
         self._keep_alive = keep_alive
+        self.state = state if state else {}
+        if 'requests_count' not in self.state:
+            self.state['requests_count'] = 0
```

Comment:
The only thing I am concerned now is that storing `requests_count` in a dict maybe overkill? Is it possible to simply that?

But I am not opposed to merge this and optimize it later.